Gruntfile.js

```diff
@@ -37,15 +37,15 @@ module.exports = function( grunt ) {
 					attrs: 'fill',
 					expand: true,
 					cwd: 'sources/svg',
-					src: ['*.svg'],
+					src: [ '*.svg', 'gutenberg/*.svg' ],
```

Comment:
Any reason for specifying explicitly both instead of using (as we do elsewhere already) the `**/*.svg` syntax? 

It feels it would make clearer when we build "everything" vs "just these subfolders".